### ISSUE TITLE
Validation sends csvs with no rows

### ISSUE THREAD

**Issue by sebastian-c** [SWS-Methodology/faoswsProduction#138]
The csvs produced by the validation sends all the csvs produced by the validation process. The output shouldn't contain empty csvs. It should look something like this:

```
|---------------Test---------------------|-invalid rows-|----------Status--------------|
1. Flag validity: Whether a flag is valid       8         .. ERROR - see flag_error.csv
2. Production range: [0, Inf)                   -         .. OK
3. Area Harvested range: [0, Inf)               -         .. OK
4. Yield range: (0, Inf)                        -         .. OK
5. Production balanced                          -         .. OK
Attached:                                       -         flag_error.csv
```

**Comment by mkao006** [comment]
This is can be a rather quick fix, but it is not a priority and also I would like to have a better implementation which is more extensible for adding new tests. Will come back to it along with issue #139 later when I have the time.

**Comment by mkao006** [comment]
resolved in 89eaafa701a023c14455bf4b4ccc2fbd54913e38
